scripts/kind-with-registry.sh

```diff
@@ -102,11 +217,11 @@ EOF
 # See: https://github.com/containerd/containerd/blob/main/docs/hosts.md
 if [ "$AZWI_ENABLED" == 'true' ]
  then
-   echo "azwi is enabled..."
+   echo "workload-identity is enabled..."
```

Comment:
I think users may be a bit confused with the change in terminology here. It might be worth including a note to set the `AZWI_ENABLED` environment variable to true